gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -339,6 +342,11 @@ gis_summary_page_constructed (GObject *object)
                             "lost forever when you restart or shutdown the computer.</b>"));
   }
 
+  /* Show the right contents for Hack images */
+  if (g_str_has_prefix (gis_driver_get_image_version (GIS_PAGE (page)->driver),
```

Comment:
This will CRITICAL if `gis_driver_get_image_version` returns NULL. In practice it will not on any Endless system that originated as an OS image, but if you (eg) debootstrap yourself a new dev environment it will be. All other places that check the image version include a NULL check so I think this one should too. That's why I think a better approach would be for driver to "parse" eos-image-version (ie split at `-`) then use `g_strcmp0()`. But if you're in a rush, just add the NULL check here I guess.

Reply:
Yeah. I guess it's better to account for `NULL` even though it's unlikely.

Reply:
Yeah. I guess it's better to account for `NULL` even though it's unlikely.